kinder/doc/kind-kinder.md

```diff
@@ -11,42 +11,41 @@ ultimately contribute back new features.
 At high level you should use [kind](https://github.com/kubernetes-sigs/kind) whenever you want
 a working Kubernetes cluster and _just_ work on it (for DEV, test, CI/CD).
 
-Instead, you should use kinder when you want to get more control on the process of creating a
-new Kubernetes cluster from "bare" machines running into a container to a working Kubernetes.
+As for kinder, you should use it when you want to have more control over the process of creating a
+new Kubernetes cluster from "bare machines" running as containers.
 
 ## Differences
 
-Kinder provider an UX designed for helping kubeadm contributors and for kubeadm E2E tests;
-Only few of the [kind](https://github.com/kubernetes-sigs/kind) commands will be available in kinder,
-because they are useful for the use cases targeted by kinder.
+Kinder provides an UX designed for helping kubeadm contributors and for kubeadm E2E tests.
+Only the subset of the [kind](https://github.com/kubernetes-sigs/kind) commands which are useful for the targeted use
+cases is available to kinder.
 
 _Building images:_
-- kinder support both containerd and docker as container runtime inside the images
-- kinder provides support for altering a base/node images and:
-     - Add a Kubernetes version to be used for `kubeadm init` (from release, CI/CD or locally build artifacts)
-     - Pre-load  TAR image files into the base/node image
-     - Replace the kubectl, kubelet, kubeadm binary to be used for `kubeadm init` (from release, CI/CD or locally
+- kinder supports both `containerd` and `docker` as container runtime inside the images
+- kinder provides support for altering base/node images by:
+     - Adding a Kubernetes version to be used for `kubeadm init` or `kubeadm upgrade` (from release, CI/CD or locally build artifacts)
+     - Pre-loading tar image files into the base/node image
+     - Replacing the kubectl, kubelet or kubeadm binary to be used for `kubeadm init` (from release, CI/CD or locally
        build artifacts)
-     - Add a Kubernetes version to be used for `kubeadm upgrade` (from release, CI/CD or locally build artifacts)
 - kinder can build images only on top of linux/amd64 base images (currently ubuntu:18.04)
 
 _Creating the cluster:_
-- kinder support both containerd and as container runtime inside the images
+- kinder support both `containerd` and `docker` as container runtime inside the images
 - kinder allows to break down the `create` operation into several atomic actions:
-    - Create machines running into a container
-    - Generate kubeadm config
-    - Generate load balancer config
+    - Creating machines running as containers
+    - Generating kubeadm config
+    - Generating load balancer config
```

Comment:
is present continuous tense really needed here?

Reply:
A few lines up I changed it to present continuous tense because it feels more natural. So I thought I should keep it consistent. But I can switch this back to how it was before if you think that makes more sense.

Reply:
ok, SGTM